go.mod

```diff
@@ -75,17 +75,19 @@ require (
 	github.com/mitchellh/mapstructure v1.4.1 // indirect
 	github.com/mitchellh/reflectwalk v1.0.0 // indirect
 	github.com/nu7hatch/gouuid v0.0.0-20131221200532-179d4d0c4d8d // indirect
+	github.com/oracle/oci-go-sdk v24.3.0+incompatible // indirect
```

Comment:
Why do we still require this older `oci-go-sdk` version when we're including `v65`? And if we do need it, why is it older than the previous `v36` that you replaced?

Reply:
Ran go mod tidy which removed older dependencies